content/applications/general/payment_acquirers/stripe.rst

```diff
@@ -14,13 +14,31 @@ Configuration
 Credentials tab
 ---------------
 
-Odoo needs your **API Credentials** to connect with your Stripe account, which comprises:
+On premise or on Odoo SH, Odoo needs your **API Credentials** to connect with your Stripe account,
+which comprises:
```

Comment:
I think it's clearer to make two sublevels titles so that people understand that those parts depend on the "version" they're using, also that way they can skip easily to what interest them

```suggestion
On-premise or Odoo SH
~~~~~~~~~~~~~~~~~~~~~

Odoo needs your **API Credentials** to connect with your Stripe account, which comprises:
```